Containerfile

```diff
@@ -21,9 +22,17 @@ LABEL \
 COPY --from=files / /
 
 RUN microdnf update --assumeyes --nodocs --setopt=keepcache=0 && \
-    microdnf install --assumeyes --nodocs --setopt=keepcache=0 tar gzip sysstat time && \
+    microdnf install --assumeyes --nodocs --setopt=keepcache=0 tar gzip time jq && \
     useradd --non-unique --uid 0 --gid 0 --shell /bin/bash notroot
 
+ARG ORAS_VERSION=1.1.0
+
+RUN curl -LO "https://github.com/oras-project/oras/releases/download/v${ORAS_VERSION}/oras_${ORAS_VERSION}_linux_amd64.tar.gz" && \
```

Comment:
Eventually... we may want to use something like a go module to pull oras in. This will ensure we'll have better provenance for this image since we would build it from source instead of taking a binary from somewhere.
(Good enough for now IMO)